src/main/java/frc/robot/Robot.java

```diff
@@ -27,7 +37,7 @@
  * creating this project, you must also update the build.properties file in 
  * the project.
  */
-public class Robot extends TimedRobot {
+public class Robot extends LoggedRobot {
```

Comment:
Does LoggedRobot extend TimedRobot?

Reply:
We should be careful that we're not missing anything by changing this inheritance

Reply:
LoggedRobot does not extend TimedRobot. Both of them implement IterativeRobotBase. However, from [the AdvantageKit installation docs](https://github.com/Mechanical-Advantage/AdvantageKit/blob/main/docs/INSTALLATION.md):
> `LoggedRobot` performs the same functions as `TimedRobot`, with some exceptions:
> - It does not support adding extra periodic functions.
> - The method `setUseTiming` allows the user code to disable periodic timing and run cycles as fast as possible during replay. The timestamp read by methods like `Timer.getFPGATimstamp()` will still match the timestamp from the real robot.
I'll check to ensure that both of these still work with our code.

Reply:
OK that probably won't be an issue

Reply:
You'll need the others to approve before you can merge

Reply:
I just checked. Changing it over will not cause errors.